Makefile

```diff
@@ -25,6 +25,7 @@ clean:
 	@rm -rf venv
 	@rm -rf venv-tools
 	@rm -rf flare_splunk_integration/bin/vendor
+	@unlink "/Applications/Splunk/etc/apps/flare_splunk_integration" || true
```

Comment:
Ah! Good call.

Reply:
Hehe I was tired of getting errors after calling the clean target